src/swish/io.ms

```diff
@@ -586,4 +587,105 @@
    'ok)
   )
 
+(isolate-mat signal-handler ()
+  (match-let*
+   ([#(EXIT #(bad-arg signal-handler sym)) (catch (signal-handler 'sym))]
+    [#(EXIT #(bad-arg signal-handler "string")) (catch (signal-handler "string" #f))]
+    [#(EXIT #(bad-arg signal-handler 123)) (catch (signal-handler SIGHUP 123))]
```

Comment:
Use SIGWINCH instead of SIGHUP since SIGHUP is trapped by default on Windows.

Reply:
Good plan.